File: rust_internal_api_system/src/grpc_key_string_maps/helloworld.rs
pub struct VoiceRequest {
}
                    
impl VoiceRequest {
    pub const uuid: &str = "uuid";
    pub const timestamp: &str = "timestamp";
    pub const voice: &str = "voice";
}
                    


pub struct VoiceReply {
}
                    
impl VoiceReply {
    pub const uuid: &str = "uuid";
    pub const timestamp: &str = "timestamp";
    pub const voice: &str = "voice";
}
                    


pub struct CurrentUsersUUIDReply {
}
                    
impl CurrentUsersUUIDReply {
    pub const uuid: &str = "uuid";
}
                    


pub struct StartSpeakingRequest {
}
                    
impl StartSpeakingRequest {
    pub const uuid: &str = "uuid";
}
                    


pub struct StopSpeakingRequest {
}
                    
impl StopSpeakingRequest {
    pub const uuid: &str = "uuid";
}
                    


pub struct HelloRequest {
}
                    
impl HelloRequest {
    pub const name: &str = "name";
}
                    


pub struct HelloReply {
}
                    
impl HelloReply {
    pub const message: &str = "message";
}

File: python_chat_with_friends_system/src/grpc_key_string_maps/helloworld.py

from typing import List

class VoiceRequest:
    uuid: str = "uuid"
    timestamp: str = "timestamp"
    voice: str = "voice"
    __property_list__: List[str] = ["uuid", "timestamp", "voice"]

class VoiceReply:
    uuid: str = "uuid"
    timestamp: str = "timestamp"
    voice: str = "voice"
    __property_list__: List[str] = ["uuid", "timestamp", "voice"]

class CurrentUsersUUIDReply:
    uuid: str = "uuid"
    __property_list__: List[str] = ["uuid"]

class StartSpeakingRequest:
    uuid: str = "uuid"
    __property_list__: List[str] = ["uuid"]

class StopSpeakingRequest:
    uuid: str = "uuid"
    __property_list__: List[str] = ["uuid"]

class HelloRequest:
    name: str = "name"
    __property_list__: List[str] = ["name"]

class HelloReply:
    message: str = "message"
    __property_list__: List[str] = ["message"]